pom.xml

```diff
@@ -29,8 +29,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <!-- DependencyGraph is final in Jenkins < 1.425 -->
-        <version>1.425</version>
+        <version>1.532.3</version>

```

Comment:
Why do you change the target Jenkins version?
I don't think you need that as far as I see your changes.